### DISCUSSION TITLE
problema

### DISCUSSION THREAD

**Discussion by nicholas20220** [iisgiua/giuaschool#311] [Richieste di aiuto] [answered]
Buonasera, come si fa a resettare il registro

**Comment by trinko** [comment]
Questa non è una segnalazione di errore. La sposto nelle richieste di aiuto.

**Comment by trinko** [comment, marked as answer]
Vedi:
[Altervista Passaggio Anno Successivo](https://github.com/iisgiua/giuaschool/discussions/303)
